db/change_cache.go

```diff
@@ -81,8 +81,8 @@ type changeCacheStats struct {
 
 func (c *changeCache) updateStats(ctx context.Context) {
 
-	c.lock.Lock()
-	defer c.lock.Unlock()
+	c.lock.RLock()
+	defer c.lock.RUnlock()
```

Comment:
Does `c._getMaxStableSequence()` require a lock because it touches `nextSequence`?

Reply:
It does, but read only

Reply:
Ah, brain mishap, looks good.